Cargo.toml

```diff
@@ -32,7 +32,7 @@ thiserror = "1.0"
 libc = "0.2.147"
 oci-spec = { version = "0.6.1", features = ["runtime"] }
 sha256 = "1.3.0"
-libcontainer = "0.1"
+libcontainer = { version = "0.1", default-features = false }
```

Comment:
is `default-features=false` needed?

Reply:
Actually not, let me remove it.

Reply:
Actually it is needed because otherwise you will get this error

```
warning: /home/mossaka/developer/runwasi/crates/containerd-shim-wasm/Cargo.toml: `default-features` is ignored for libcontainer, since `default-features` was not specified for `workspace.dependencies.libcontainer`, this could become a hard error in the future
```